unified_targets/docs/Manufacturers.md

```diff
@@ -3,17 +3,21 @@
 This is the official list of manufacturer ids (`manufacturer_id` in the target config) that will be supported for loading onto unified targets by Betaflight configurator.
 
 
-|Manufacturer Id|Name|Contact|
-|-|-|-|
-|CUST|'Custom', to be used for homebrew targets||
-|AFNG|AlienFlight NG|https://www.alienflightng.com/|
-|AIRB|Airbot|https://store.myairbot.com/|
-|BKMN|Jason Blackman|https://github.com/blckmn|
-|DRCL|dronercland|https://www.instagram.com/dronercland/|
-|DYST|DongYang Smart Technology Co.,Ltd (dys)|http://www.dys.hk/|
-|FFPV|Furious FPV|https://furiousfpv.com/|
-|HAMO|Happymodel|http://www.happymodel.cn/|
-|HBRO|Holybro|http://www.holybro.com/index.html|
-|MTKS|Matek Systems|http://www.mateksys.com/|
-|RCTI|RCTimer|http://rctimer.com/|
-|SPRO|Seriously Pro Racing (SP Racing)|http://seriouslypro.com/|
+| Manufacturer Id |                   Name                    |                Contact                 |
+| :-------------: | :---------------------------------------: | :------------------------------------: |
+|      CUST       | 'Custom', to be used for homebrew targets |                                        |
+|      AFNG       |              AlienFlight NG               |     https://www.alienflightng.com/     |
+|      AIRB       |                  Airbot                   |      https://store.myairbot.com/       |
+|      BKMN       |              Jason Blackman               |       https://github.com/blckmn        |
+|      DRCL       |                dronercland                | https://www.instagram.com/dronercland/ |
+|      DYST       |  DongYang Smart Technology Co.,Ltd (dys)  |           http://www.dys.hk/           |
+|      FFPV       |                Furious FPV                |        https://furiousfpv.com/         |
+|      HAMO       |                Happymodel                 |       http://www.happymodel.cn/        |
+|      HBRO       |                  Holybro                  |   http://www.holybro.com/index.html    |
+|      MTKS       |               Matek Systems               |        http://www.mateksys.com/        |
+|      RCTI       |                  RCTimer                  |          http://rctimer.com/           |
+|      SPRO       |     Seriously Pro Racing (SP Racing)      |        http://seriouslypro.com/        |
+|      IFRC       |                IFLIGHT RC                 |      https://www.iflight-rc.com/       |
+|      KKRC       |                Kingkong RC                |         https://www.ldarc.com/         |
+|      GEPR       |                   GEPRC                   |         https://www.geprc.com/         |
+|      EXFR       |                 EXUAV RC                  |    https://www.facebook.com/EXUAV/     |
```

Comment:
I think they must be in alphabetical order...

Reply:
Okay. I will fix it.